File: onyx-database-tests/src/pojo/TransientValue.java
package pojo;

import java.io.Serializable;
import java.util.Date;

/**
 * Created by timothy.osborn on 4/14/15.
 */
public class TransientValue implements Serializable
{
    public TransientValue()
    {

    }

    public int intValue;

    public transient long longValue;

    public Date zdateValue;
}

File: onyx-database-tests/src/main/kotlin/pojo/TransientValue.kt
package pojo

import java.io.Serializable
import java.util.Date

/**
 * Created by timothy.osborn on 4/14/15.
 */
class TransientValue : Serializable {

    var intValue: Int = 0

    @Transient
    var longValue: Long = 0

    var zDateValue: Date? = null
}
